newIDE/app/src/VariablesList/index.js

```diff
@@ -149,7 +144,11 @@ export default class VariablesList extends React.Component<Props, State> {
   ) => {
     const { variablesContainer, inheritedVariablesContainer } = this.props;
 
-    if (inheritedVariablesContainer && origin === 'parent') {
+    if (
+      inheritedVariablesContainer &&
+      origin === 'parent' &&
+      typeof name !== 'number'
```

Comment:
Generally seeing typeof looks a bit like a code smell. Especially, given that the typing tells us it's a string :) Maybe you need to inspect something else like the type of the variable? Or pass a new argument?